Sources/Database/Live.swift

```diff
@@ -778,6 +785,24 @@ extension Client {
           "stripe_payment_intent_status" character varying NOT NULL DEFAULT '\(raw: PaymentIntent.Status.requiresPaymentMethod.rawValue)'
           """
         )
+        try await database.run(
+          """
+          ALTER TABLE "episode_progresses"
+          ADD COLUMN IF NOT EXISTS
+          "is_finished" boolean NOT NULL DEFAULT FALSE
+          """
+        )
+        try await database.run(
+          """
+          DROP INDEX IF EXISTS "index_episode_progresses_on_episode_sequence_user_id"
+          """
+        )
+        try await database.run(
+          """
+          CREATE UNIQUE INDEX IF NOT EXISTS "index_episode_progresses_on_user_id_episode_sequence"
+          ON "episode_progresses" ("user_id", "episode_sequence")
+          """
```

Comment:
Re-indexing `episode_progresses` so that `user_id` is first to be more efficient.

Reply:
Would be cool to have a schema version sometime to scope these migrations, but prob would wanna revisit databases from scratch first 😅